### PR TITLE
Wiki/sidebar update

### PR DESCRIPTION
**Pull request by RdkLwg**
## Description
Updates regarding wiki sidebar

## Motivation and Context
#383

## Screenshots (if appropriate)
n/a

## Upgrade notes (if appropriate)
n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [x] I have reviewed (and updated if needed) the documentation regarding this change

---
I hereby agree to the terms of the AET Contributor License Agreement.

### CODE REVIEW

File: documentation/src/main/wiki/_Sidebar.md [hunk 1]
```diff
@@ -9,36 +9,34 @@
...
         * [[Suite Structure|SuiteStructure]]
-        * [[Best practices|BestPractices]]
+        * [[Best Practices|BestPractices]]
     * [[Running Suite|RunningSuite]]
         * [[Tracking Progress|TrackingProgress]]
     * [[Features|Features]]
         * [[Open|Open]]
         * [[Collectors|Collectors]]
             * [[Accessibility|AccessibilityCollector]]
-            * [[Client Side Performance|ClientSidePerformanceCollector]]
```
Comment (wiiitek) [inline documentation/src/main/wiki/_Sidebar.md]: We don't want to have this in TOC ?

Comment (RdkLwg) [reply documentation/src/main/wiki/_Sidebar.md]: Nope. It no longer works. ySlow is no longer supported by Chrome and the last github commit took place 5 years ago.

Comment (wiiitek) [reply documentation/src/main/wiki/_Sidebar.md]: ok :(

Comment (RdkLwg) [reply documentation/src/main/wiki/_Sidebar.md]: There is the idea to add audits generated with Lighthouse. A corresponding task has been raised at #368. Another ticket to remove redundant code related to ySlow has been raised at #481.

**Pull request merged**